ephios/plugins/questionnaires/templates/questionnaires/savedanswer_list.html

```diff
@@ -0,0 +1,42 @@
+{% extends "core/settings/settings_base.html" %}
+{% load i18n %}
+{% load answer_text %}
+
+{% block title %}
+    {% translate "Saved questionnaire answers" %}
+{% endblock %}
+
+{% block settings_content %}
+    <h3>
+        {% translate "Saved questionnaire answers" %}
+    </h3>
+
+    <ul class="list-group mb-3">
+        {% for answer in savedanswer_list %}
+            <li class="list-group-item">
+                <div class="row mb-0">
+                    <div class="col-12 col-lg-5">
+                        <h5 class="mb-0 d-inline">
+                            {{ answer.question.name }}
+                        </h5>
+                        <br/>
+                        <span class="text-muted">{{ answer.question.question_text }}</span>
+                    </div>
+                    <div class="col-12 col-lg-5">
+                        {{ answer.answer | answer_text }}
+                    </div>
+                    <div class="col-12 col-lg-auto d-flex align-items-end align-items-lg-center justify-content-end">
```

Comment:
i think the buttons could go to the right side on large screens, the space seems a bit wasted atm

Reply:
What do you mean? On a sufficiently large screen, the buttons are on the right:

<img width="1320" height="541" alt="image" src="https://github.com/user-attachments/assets/de7bf498-c1ba-439e-9b39-fe07a2580e7b" />

Reducing the breakpoint from `lg` to `md` does not help, since the column of the buttons is to wide to fit on the right, so there is still a line break. And I would rather not make the columns for the question and answer smaller, since both texts may be longer than just a few characters…

Reply:
ah nevermind this seems to have changed with a more recent commit